core/tests/data/unicode_and_str_handler.py

```diff
@@ -0,0 +1,32 @@
+# coding: utf-8
+#
+# Copyright 2019 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""This is a sample file for testing unicode and str characters.
+
+Unless the strings are explicitly wrapped with b'', they would denote unicode,
+otherwise str.
+"""
+
+from __future__ import unicode_literals  # pylint: disable=import-only-modules
```

Comment:
Why are we disabling import-only-modules throughout the PR? This import `import __future__` works fine, right?

Reply:
No this doesn't work fine for future imports as they are not modules. You can look at [this](https://github.com/oppia/oppia/pull/6613#discussion_r280044543) comment for details. Thanks
> I mean that absolute_import, print_function, etc are not modules. The import statement here is more of a "Future" statement -- they change how the Python module is parsed, which is why they must be at the top of the file. They give new or different meaning to words or symbols in the file. This is why we can not use the functions from the future module if its imported at module level. If you know of any way to do it, please tell me. Thanks

Reply:
Oh, agreed then -- let's follow this convention. Thanks for the link!